bdb/bdb_verify.c

```diff
@@ -93,8 +93,8 @@ int bdb_verify(
 
     BDB_READLOCK("bdb_verify");
 
-    if (rc = bdb_state->dbenv->lock_id_flags(bdb_state->dbenv, &lid,
-                                             DB_LOCK_ID_READONLY)) {
+    if ((rc = bdb_state->dbenv->lock_id_flags(bdb_state->dbenv, &lid,
+                                             DB_LOCK_ID_READONLY))) {
```

Comment:
By itself `if(rc)` is more readable. We're not discussing that case though. Its about stuff like this one.. By the time the expression finishes, there are 3 parens closing. I prefer if this were: `if ((rc = bdb_state->dbenv->lock_id_flags(bdb_state->dbenv, &lid, DB_LOCK_ID_READONLY)) != 0)`.

Reply:
ok, applying.